packages/arb-token-bridge-ui/src/hooks/TransferPanel/useGasSummary.ts

```diff
@@ -67,14 +67,16 @@ export function useGasSummary(): UseGasSummaryResult {
       walletAddress,
       sourceChainId: networks.sourceChain.id,
       destinationChainId: networks.destinationChain.id,
-      amount: amountBigNumber,
+      amount:
```

Comment:
this would be very annoying to use if we were to reuse the hook in the future, feel like we can abstract all the props inside the hook itself and only use:
```typescript
const { gasEstimates: estimateGasResult, error: gasEstimatesError } = useGasEstimates()
```